### DISCUSSION TITLE
✅ 🔗 AVIF Support with Lightbox (Duplicate of #25175)

### DISCUSSION THREAD

**Discussion by stefanbiba** [elementor/elementor#28169] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Pictures in AVIF format cannot be opened with Lightbox. Instead the picture is open itself.

### Describe the solution you'd like.

Picture open with Lightbox

### Additional context

AVIF has better compression than WEBP (which works great with Lightbox) and is supported by all modern browsers.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by Angelo-Verona** [comment]
same as #25175

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementorv3.29.0! 🥳 

Elementor's Lightbox now supports **AVIF** image formats natively!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
